src/stories/buttonStories.js

```diff
@@ -10,198 +10,200 @@ storiesOf("Button", module)
+      type="btnError"
     />
   ))
   .add("error outline", () => (
     <Button
       label="Button"
       onClick={action("click")}
-      className="btn btn-error-outline"
+      type="btnErrorOutline"
     />
   ))
   .add("error-light", () => (
     <Button
       onClick={action("click")}
       label="Button"
-      className="btn btn-error-light"
+      type="btnErrorLight"
     />
   ))
   .add("error with gray shadow", () => (
     <Button
       label="Button"
       onClick={action("click")}
-      className="btn btn-error-gray-shadow"
+      type="btnErrorGrayShadow"
     />
   ))
   .add("error with shadow", () => (
     <Button
       label="Button"
       onClick={action("click")}
-      className="btn btn-error-shadow"
-    />
-  ))
-  // devider Button
-  .add("divider", () => (
-    <Button
-      onClick={action("click")}
-      label="Button"
-      className="btn btn-devider"
-    />
-  ))
-  .add("devider outline", () => (
-    <Button
-      label="Button"
-      onClick={action("click")}
-      className="btn btn-devider-outline"
-    />
-  ))
-  .add("devider-light", () => (
-    <Button
-      onClick={action("click")}
-      label="Button"
-      className="btn btn-devider-light"
-    />
-  ))
-  .add("devider with gray shadow", () => (
-    <Button
-      label="Button"
-      onClick={action("click")}
-      className="btn btn-devider-gray-shadow"
-    />
-  ))
-  //rounded buttons
-  .add("button rounded corners with 5px", () => (
-    <Button
-      onClick={action("click")}
-      label="Button"
-      className="btn btn-corner-5px"
-    />
-  ))
-  .add("button rounded corners with 200px", () => (
-    <Button
-      onClick={action("click")}
-      label="Button"
-      className="btn btn-corner-200px"
-    />
-  ))
-  .add("chip", () => (
-    <Button onClick={action("click")} label="Chip" className="btn btn-chip" />
-  ))
-  //ellipse Button
-  .add("ellipse", () => (
-    <Button
-      onClick={action("click")}
-      label="+"
-      className="btn btn-primary ellipse"
-    />
-  ))
-  .add("ellipse outline", () => (
-    <Button
-      label="+"
-      onClick={action("click")}
-      className="btn btn-primary-outline ellipse"
-    />
-  ))
-  .add("ellipse-light", () => (
-    <Button
-      onClick={action("click")}
-      label="+"
-      className="btn btn-primary-light ellipse"
-    />
-  ))
-  .add("ellipse with gray shadow", () => (
-    <Button
-      label="+"
-      onClick={action("click")}
-      className="btn btn-primary-gray-shadow ellipse"
-    />
-  ))
-  .add("ellipse with shadow", () => (
-    <Button
-      label="+"
-      onClick={action("click")}
-      className="btn btn-primary-shadow ellipse"
-    />
-  ))
-  //secondary with vector
-  .add("secondary with vector", () => (
-    <Button
-      label="+ Button"
-      onClick={action("click")}
-      className="btn btn-secondary"
-    />
-  ))
-  //
+      type="btnErrorShadow"
+    />
+  ))
+  // // devider Button
```

Comment:
Can we remove this comment?

Reply:
I removed the duplicated forward slash but left the rest of comment because we need the comments to make it easy for later search because of different button types we divided the buttons and marked them by the comments